packages/ddp-rate-limiter/ddp-rate-limiter-tests.js

```diff
@@ -77,6 +77,68 @@ testAsyncMulti("ddp rate limiter - matchers get passed correct arguments", [
   }
 ]);
 
+testAsyncMulti("ddp rate limiter - callbacks get passed correct arguments", [
+  function (test, expect) {
+    _.extend(this, createTestUser(test, expect));
+  },
+  function (test, expect) {
+    var self = this;
+    Meteor.call("addRuleToDDPRateLimiter", expect(function(error, result) {
+     self.ruleId = result;
+    }));
+  },
+  function (test, expect) {
+    Meteor.call('dummyMethod', function() {});
```

Comment:
I think we don't need to because these calls do not expect any results from server. Are there any possible errors?